packages/cli/src/ui/components/__snapshots__/IDEContextDetailDisplay.test.tsx.snap

```diff
@@ -0,0 +1,24 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`IDEContextDetailDisplay > handles duplicate basenames by showing path hints 1`] = `
+"
+╭──────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ VS Code Context (ctrl+e to toggle)                                                               │
+│                                                                                                  │
+│ Open files:                                                                                      │
+│ - bar.txt (/foo) (active)                                                                        │
```

Comment:
minor:
@miguelsolorio any thoughts on simple ways to polish this UX for showing files with the same name but different directories a bit clearer?
My minor concern with the current ux is it is a bit odd to have the directory after the file name and I don't really like the multiple adjacent blocks (in parens).